fabtests/pytest/conftest.py

```diff
@@ -144,9 +145,15 @@ def _populate_normal_command(self, command, host_type):
         if host_type == "host":
             return command
 
+        if self.oob_address_exchange:
+            oob_argument = "-E"
+            if "PYTEST_XDIST_WORKER" in os.environ:
+                oob_port = 9228 + int(os.environ["PYTEST_XDIST_WORKER"].replace("gw", ""))
```

Comment:
what is `gw`?

Reply:
Not exactly sure. pytest-xdist set the environment variable value to `gw0`, `gw1`, ...

Reply:
It seems there is a `worker_id` fixture for the same usage.